Optimize assets.

K0-RR opened the pull request:
Lossless compression.

K0-RR committed:
Reduce!

ghost commented:
What exactly is this doing to the assets can you give me like a weighing of the pros and cons and a more in-depth understanding of what's happening here?

K0-RR commented:
I used oxipng for this EDIT -corrected the command `oxipng -arvZt8 -o max --strip all --fix`.
- pros
smaller jar size.
- cons
I don't see any.
>more in-depth understanding of what's happening here

Link to the tool repo - https://github.com/shssoichiro/oxipng. Here is a part of a log from my other optimization:
```
Processing: /home/username/Documents/GitHub/REPO/src/main/resources/assets/modid/textures/item/file.png
    16x16 pixels, PNG format
    4x8 bits/pixel, RGBA
    IDAT size = 463 bytes
    File size = 2279 bytes
Reducing image to 8 bits/pixel, 95 colors in palette
Trying: 6 combinations
Found better combination:
    zc = 0  zs = 0  f = 0        126 bytes
    IDAT size = 126 bytes (337 bytes decrease)
    file size = 514 bytes (1765 bytes = 77.45% decrease)
```

This does not reduce the quality of the files, only the size. You can use this Github bot http://imgbot.net/ to achieve similar results to mine (tbh the results are worse but the bot does not require any knowledge or manual to use, it just opens ready-to-merge prs).

K0-RR commented:
I also opened PRs in Your other mods. Feel free to do the compression by yourself.
https://github.com/TerraformersMC/Campanion/pull/147
https://github.com/TerraformersMC/Traverse/pull/74
https://github.com/TerraformersMC/Terrestria/pull/259

I changed the tool/command recently to get better results so I updated it here above in case someone wants it.
